<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tasksDialog</class>
 <widget class="QDialog" name="tasksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>318</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>615</width>
    <height>318</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>615</width>
    <height>318</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Задача</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainResource.qrc">
    <normaloff>:/images/Tasks64.png</normaloff>:/images/Tasks64.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_content">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>423</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Срок:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="m_datePlan">
      <property name="minimumDate">
       <date>
        <year>2001</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Дата завершения:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="m_dateFact">
      <property name="minimumDate">
       <date>
        <year>2001</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>581</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Время по плану:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_planTime">
      <property name="inputMask">
       <string>000:00</string>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Затраченное время:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_factTime">
      <property name="inputMask">
       <string>000:00</string>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>118</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>11</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_name</tabstop>
  <tabstop>m_content</tabstop>
  <tabstop>m_datePlan</tabstop>
  <tabstop>m_dateFact</tabstop>
  <tabstop>m_planTime</tabstop>
  <tabstop>m_factTime</tabstop>
 </tabstops>
 <resources>
  <include location="MainResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tasksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tasksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
